core/templates/pages/creator-dashboard-page/suggestion-modal-for-creator-view/suggestion-modal-for-creator-view.service.ts

```diff
@@ -20,22 +20,23 @@ require('components/ck-editor-helpers/ck-editor-4-widgets.initializer.ts');
 
 require('domain/utilities/url-interpolation.service.ts');
 require('services/suggestion-modal.service.ts');
+// eslint-disable-next-line max-len
+require('pages/creator-dashboard-page/suggestion-modal-for-creator-view/suggestion-modal-for-creator-view.controller');
```

Comment:
You could break this across multiple lines right? And remove the eslint disable?

Reply:
Done.